lib/core/test/unit/Cardano/Wallet/Api/Malformed.hs

```diff
@@ -912,48 +912,61 @@ instance Malformed (BodyParam WalletPutPassphraseData) where
                 { "old_passphrase": #{wPassphrase}
                 , "new_passphrase" : 100
                 }|]
-              , "Error in $['new_passphrase']: parsing Passphrase failed, expected String, but encountered Number"
+              , "Error in $['new_passphrase']: parsing Passphrase failed, expected String, but encountered Number, OldPassphrase variant"
```

Comment:
It is redundant in this specific case, but it's coming from the _Variant.hs_ general sum type parsing implementation. 
From the parsing perspective, it is good that we know which variant was chosen, so that the parsing failure has more context.
You can imagine complex sum types where the variant selection is not that obvious. Like matching multiple subsets of the keys.